apis/che-controller/v1alpha1/component_types.go

```diff
@@ -16,23 +16,19 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-type RoutingType string
-
-const (
-	SingleHost RoutingType = "singlehost"
-	MultiHost  RoutingType = "multihost"
-)
-
 // CheManagerSpec holds the configuration of the Che controller.
 // +k8s:openapi-gen=true
 type CheManagerSpec struct {
-	// The hostname to use for creating the workspace endpoints
-	// This is used as a full hostname in the singlehost mode. In the multihost mode, the individual
-	// endpoints are exposed on subdomains of the specified host.
-	Host string `json:"host,omitempty"`
+	// GatewayHost is the full host name used to expose workspace endpoints that have relocatable roots.
+	// Mandatory on Kubernetes, optional on OpenShift.
+	GatewayHost string `json:"gatewayHost,omitempty"`
 
-	// Routing defines how the Che Router exposes the workspaces and components within
-	Routing RoutingType `json:"routing,omitempty"`
+	// GatewayDisabled enables or disables routing of the relocatable workspace compontents through a common gateway.
+	// Default value is "false" meaning that the gateway is enabled.
+	// If disabled, all endpoints are deployed on a subdomain of a WorkspaceComponentBaseDomain. If enabled, components marked
```

Comment:
I found this confusing. Does `If disabled` means that it is set to true (so gateway is disabled), or false ("property" is disabled)?
I think wording "If set to true, ..." would be more clear for the reader.